docs/UserGuide.md

```diff
@@ -152,11 +152,11 @@ The table below illustrates the meaning of the prefixes and their respective inp
 | Prefix | Meaning                  | Input       | Constraints |
 |:-------|:-------------------------|:------------|:------------|
 |   -    |                          | INDEX       | Index of applicant specified must not be more than the total number of applicants in the list. |
-| **n/** | Applicant's Name         | NAME        |
-| **p/** | Applicant's Phone Number | PHONE_NUMBER|
-| **e/** | Applicant's Email        | EMAIL       |
-| **j/** | Job Applied              | JOB         |
-| **r/** | Application Round        | ROUND       |
+| **n/** | Applicant's Name         | NAME        | Names can only contain Unicode characters, and cannot contain any numbers. Name's can contain the following special characters: `-` and`'`. **An applicant's name is unique** i.e. two applicants cannot have the same name.
```

Comment:
Maybe elaborate on what is Unicode? Or use a different way of expressing it? Most laymen would not be aware of what is Unicode or UTF-8

Reply:
Changed to a more simple explanation